packages/eslint-plugin/src/util/referenceContainsTypeQuery.ts

```diff
@@ -0,0 +1,20 @@
+import type { TSESTree } from '@typescript-eslint/utils';
+import { AST_NODE_TYPES } from '@typescript-eslint/utils';
+
+/**
+ * Recursively checks whether or not a given reference has a type query declaration among it's parents
```

Comment:
[Docs] An unnecessary _"or not"_, and an unnecessary apostrophe:

```suggestion
 * Recursively checks whether a given reference has a type query declaration among its parents
```

Reply:
same as above, copy-pasted
fixed :)